ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -213,6 +213,26 @@
 {%         endif %}
 | {{ evpn_hostflap_detection_state }} | {{ evpn_hostflap_detection_window }} |  {{ evpn_hostflap_detection_threshold }} |
 {%     endif %}
+{%     set evpn_l2_gw_config = namespace(peer_groups = [], configured = false) %}
+{%     for peer_group in router_bgp.peer_groups | arista.avd.natural_sort %}
+{%         if router_bgp.address_family_evpn.peer_groups[peer_group].domain_remote is arista.avd.defined(true) %}
+{%             do evpn_l2_gw_config.peer_groups.append(peer_group) %}
+{%         endif %}
+{%     endfor %}
+{%     if evpn_l2_gw_config.peer_groups | length > 0 %}
+
+#### EVPN DCI Gateway Summary
+
+| Settings | Value |
+| -------- | ----- |
+| Remote Domain Peer Groups | {{ evpn_l2_gw_config.peer_groups | join(", ") }} |
+{%         if router_bgp.address_family_evpn.neighbor_default.next_hop_self_received_evpn_routes.enable is arista.avd.defined(true) %}
+| L3 Gateway Configured | True |
```

Comment:
I think you oversimplified this with your last commit. This line (and the next) is about L3 gateway, but you never get here unless you also have L2 gateway configured.
Remove the last commit and and another check for these keys in the section above the table, to set the "configured" flag.

Reply:
Added back the logic and also set configured flag to true based on l3 gw settings.